www/front_src/src/Resources/Details/Header.tsx

```diff
@@ -41,22 +45,35 @@ interface MakeStylesProps {
 }
 
 const useStyles = makeStyles<Theme, MakeStylesProps>((theme) => ({
+  border: {
+    borderLeft: `${theme.spacing(1.5 / 8)} dashed`,
```

Comment:
```suggestion
    borderLeft: `1px dashed`,
```
You can use the px format here.

Imo, I think it's better to use the Divider component : https://mui.com/material-ui/react-divider/#list-dividers